config/routes.rb

```diff
@@ -1,6 +1,9 @@
 FinderFrontend::Application.routes.draw do
   mount GovukPublishingComponents::Engine, at: "/component-guide"
 
+  get "/sign-in", to: proc { [200, {}, %w[OK]] }
+  get "/sign-out", to: proc { [200, {}, %w[OK]] }
```

Comment:
What are these for?

Reply:
From what I found if you remove or comment these out then we cannot run feature tests,
The routes at the bottom of this file seem to do a very generous wildcard match (I'm not sure why I guess there's an open ended number of finder slugs?).

But without this proc it things `/sign-in` is a finder and treats it as a content item finder needs to process.
These stub those out, but if you typed the real URL in on prod, the router takes over and throws you to `frontend`.

I couldn't figure out a better way to test these and still get routes helpers.
I tried `direct` too and none, but no joy.

Open to better ideas?

Reply:
Can we make the tests not follow the redirects?  I don't like the idea of adding dummy `/sign-in` and `/sign-out` routes to every frontend app which uses accounts.